src/chainstate/stacks/db/mod.rs

```diff
@@ -223,6 +223,7 @@ impl DBConfig {
                 self.version == "2" || self.version == "3" || self.version == "4"
             }
             StacksEpochId::Epoch21 => self.version == "3" || self.version == "4",
+            StacksEpochId::Epoch22 => self.version == "4",
```

Comment:
I'm not sure about this change, what versions should be allowed.

Reply:
@kantai can you chime in on this point? Otherwise, I think this should be ready to merge.